app/facades/food_facade.rb

```diff
@@ -0,0 +1,11 @@
+class FoodFacade
+    def self.search(keyword)
+        data = FoodService.search(keyword)[:foods]
+        limit_10 = data[0..10]
+        limit_10.map { |food| Food.new(food)}
+    end
+
+    def self.search_results(keyword)
+        data = FoodService.search(keyword)[:totalHits]
+    end
+end
```

Comment:
In a follow up from my last comment, you might consider just having the `search` method, and letting that method return both the collection of food objects as well as the total hits.